packages/client/src/raffles/types/raffles.types.ts

```diff
@@ -0,0 +1,49 @@
+export enum RAFFLE_STATUS {
+  DRAFT = 'Draft',
+  READY = 'Ready',
+  ACCEPTION_ENTRIES = 'AcceptingEntries',
+  ENTRIES_CLOSED = 'EntriesClosed',
+  WINNER_SELECTION = 'WinnersSelection',
+  DRAW_CLOSED = 'DrawClosed',
+  CLOSED = 'Closed',
+}
+export type Raffle = {
```

Comment:
Keep the empty line above here and in line 38 please 😇

Reply:
done